docs/src/components/_overview.md

```diff
@@ -1,10 +1,33 @@
-# High-level overview
+> 🚧 Valence protocol architecture and developer documentation is still evolving rapidly. Portions of the toolchain have stabilized to build cross-chain vaults, and extending vaults with multi-party agreements. Send us a message on [X](https://x.com/valencezone) if you'd like to get started!
 
-This section provides a high-level breakdown of the components that compose a Valence cross-chain program.
+# Introduction
+
+**Valence** is a unified development environment that enables building *trust-minimized cross-chain DeFi applications*, called **Valence programs**.
+
+Valence programs are:
+
+- **Easy to understand** and **quick to deploy**: a program can be set up with a configuration file and no code.
+- **Extensible**: if we don't support a DeFi integration out of the box, you can write one yourself in a matter of hours!
+
+> **Example Use-case**:
+>
+> A DeFi protocol wants to bridge tokens to another chain and deposit them into a vault. After a certain date, it wants to unwind the position. While the position is active, It may also want to delegate the right to change vault parameters to a designated committee, as long as the parameters are within a certain range.
```

Comment:
```suggestion
> A DeFi protocol wants to bridge tokens to another chain and deposit them into a vault. After a certain date, it wants to unwind the position. While the position is active, it may also want to delegate the right to change vault parameters to a designated committee, as long as the parameters are within a certain range.
```